pkg/v3/types/basetypes.go

```diff
@@ -14,6 +15,7 @@ const (
 var checkResultStringTemplate = `{
 	"PipelineExecutionState":%d,
 	"Retryable":%v,
+	"RetryInterval":%d,
```

Comment:
is there a way to comment here to specify this is in seconds?

Reply:
+1

Reply:
We can comment above this string, but then maybe to explain other fields as well
P.S. I don't like this way of encoding (with string template) in any case, planning to remove it at some point.